src/pages/board-enrollment/student-detail/[userId].tsx

```
@@ -273,9 +290,7 @@ const BoardEnrollmentDetail = () => {
   const handleNext = async () => {
     setFormDataUpdated(false);
   
-    // Calculate the next step value first
-    let nextStep = activeStep;
-    nextStep = nextStep < 4 ? nextStep + 1 : nextStep;
+    let nextStep = activeStep < 4 ? activeStep + 1 : activeStep;
```

ESLint

[error] 'nextStep' is never reassigned. Use 'const' instead.

(prefer-const)